test/meson.build

```diff
@@ -33,6 +33,12 @@ test('khronos_typedefs',
                 c_args: common_cflags,
                 dependencies: libepoxy_dep,
                 include_directories: libepoxy_inc))
+test('gl_version',
```

Comment:
We'll also need a change for `Makefile.am`, at least until we drop Autotools support.